x-pack/plugins/upgrade_assistant/server/routes/ml_snapshots.ts

```diff
@@ -310,6 +314,9 @@ export function registerMlSnapshotRoutes({ router, lib: { handleEsError } }: Rou
               });
             }
 
+            log.error(
```

Comment:
Since both `customError` messages are the same in this route, I've added an extra error log so that if this error ever comes up, we could tell from which of the two branches it came from and what response caused it. This shouldn't happen if everything worked successfully, so it shouldn't cause log spam for users and will make it easier for us to debug this kind of problems.